src/Faker/Provider/tk_TM/Person.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Faker\Provider\tk_TM;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @var array Turkmen person name formats.
+     */
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{firstNameFemale}} {{lastName}}a',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}a',
+    );
+
+    /**
+     * @var array Turkmen first names.
+     */
+    protected static $firstNameMale = array(
```

Comment:
I could add a lot more names of course, but there is no source for names in my language to my knowlegde because I have added these from my head. Finding a source might be a problem, but I will keep adding more data.